surveys/2024-annual-survey/translations/de.md

```diff
@@ -391,14 +391,14 @@ Columns:
 - Ich wusste nicht, dass dies stabilisiert wurde
 - Ich weiß nicht, was das ist
 
-### Wie kompilieren Sie Ihre Rust Projekte?
+### Wie bauen Sie Ihre Rust Projekte?
```

Comment:
English version also says "build" (German "bauen"), I guess that covers more than compilation, e.g. also generating documentation